js/Gantt/Customizations.md

```diff
@@ -250,7 +250,7 @@ $(function () { $("#GanttContainer").ejGantt({ //… taskbarTooltipTemplateId: "
 
 The following screenshot shows Gantt with task tooltip customization.
 
-![](/js/Gantt/Customization_images/Customization_img3.png)
+![Taskbar tooltip in JavaScript Gantt](/js/Gantt/Customization_images/Customization_img3.png)
```

Comment:
Task tooltip customization in JavaScript Gantt. Check all image alt and name correctly